content/source/docs/cloud/run/cli.html.md

```diff
@@ -204,3 +204,19 @@ FALSE - my-policy.sentinel:1:1 - Rule "main"
 
 Error: Organization policy check hard failed.
 ```
+
+## Targeted Plan and Apply
+
+-> **Version note:** Targeting support was added client-side in Terraform v0.12.26 and also requires server-side support that may not be available for all Terraform Enterprise deployments yet.
```

Comment:
If you'd like to be more specific about TFE support, the next version that will contain our server-side changes is `v202006-1`!

Reply:
I pondered this a bit and it felt weird to me to include that given that this release is not so imminent and so there's more room for things to shift here prior to release and for us to forget to come back and update this. As a compromise, I'm thinking maybe we leave this vague for now and then we can consider updating it once we hit the `v202006-1` feature freeze. There's still some risk of us forgetting to update it at the freeze, but at least in that case the more vague statement I wrote here will remain true. Does that seem reasonable? If you disagree I'm not opposed to making a follow-up commit to update it now.

Reply:
That makes sense to me - better vague than misleading, in the case that something changes between now and June!